pkg/sso/client_test.go

```diff
@@ -33,6 +34,14 @@ func TestClientAuthorizeURL(t *testing.T) {
 			},
 			expected: "https://api.workos.com/sso/authorize?client_id=proj_123&domain=lyft.com&redirect_uri=https%3A%2F%2Fexample.com%2Fsso%2Fworkos%2Fcallback&response_type=code&state=custom+state",
 		},
+		{
+			scenario: "generate url with state",
```

Comment:
Should rename this scenario by `generate url with provider`.

Reply:
👍